tests/iterators_tests/test_balanced_serial_iterator.py

```diff
@@ -1,11 +1,27 @@
 import numpy
 import pytest
-from chainer_chemistry.datasets.numpy_tuple_dataset import NumpyTupleDataset
 
+from chainer import serializer
+
+from chainer_chemistry.datasets.numpy_tuple_dataset import NumpyTupleDataset
 from chainer_chemistry.iterators.balanced_serial_iterator import BalancedSerialIterator  # NOQA
 from chainer_chemistry.iterators.balanced_serial_iterator import IndexIterator  # NOQA
 
 
+class DummySerializer(serializer.Serializer):
+
+    def __init__(self, path=[]):
```

Comment:
Use `()`, which is immutable as a default value rather than `[]`.  See e.g. [here](https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html).